imap_processing/tests/spice/test_kernels.py

```diff
@@ -159,12 +175,77 @@ def test_et_times(pointing_frame_kernels):
     """Tests get_et_times function."""
     spice.furnsh(pointing_frame_kernels)
 
-    file, _, _, _ = spice.kdata(0, "ck")
-    et_start, et_end, et_times = _get_et_times(file)
+    ck_kernel, _, _, _ = spice.kdata(0, "ck")
+    ck_cover = spice.ckcov(ck_kernel, -43000, True, "INTERVAL", 0, "TDB")
+    et_start, et_end = spice.wnfetd(ck_cover, 0)
+    et_times = _get_et_times(et_start, et_end)
 
-    assert et_start == 802008069.184905
-    assert et_end == 802015267.184906
     assert et_times[0] == et_start
     assert et_times[-1] == et_end
 
     return et_times
+
+
+@ensure_spice
+def test_multiple_attempts(pointing_frame_kernels, tmp_path):
+    """Tests create_pointing_frame function with multiple pointing kernels."""
+    spice.furnsh(pointing_frame_kernels)
+
+    # Check that a single segment is added regardless of how many times
+    # create_pointing_frame is called.
+    create_pointing_frame(pointing_frame_path=tmp_path / "imap_dps.bc")
+    ck_cover = spice.ckcov(
+        str(tmp_path / "imap_dps.bc"), -43901, True, "INTERVAL", 0, "TDB"
+    )
+    num_intervals = spice.wncard(ck_cover)
+    assert num_intervals == 1
+
+    create_pointing_frame(pointing_frame_path=tmp_path / "imap_dps.bc")
+    ck_cover = spice.ckcov(
+        str(tmp_path / "imap_dps.bc"), -43901, True, "INTERVAL", 0, "TDB"
+    )
+    num_intervals = spice.wncard(ck_cover)
+    assert num_intervals == 1
+
+
+@ensure_spice
+def test_multiple_pointings(multiple_pointing_kernels, spice_test_data_path):
+    """Tests create_pointing_frame function with multiple pointing kernels."""
+    spice.furnsh(multiple_pointing_kernels)
+
+    create_pointing_frame(
+        pointing_frame_path=spice_test_data_path / "imap_pointing_frame.bc"
```

Comment:
Wait, why are you creating the pointing frame in the `imap_test_data_path`? This should be going into a temp directory. 
```suggestion
        pointing_frame_path=imap_data_access.config["DATA_DIR"] / "imap_pointing_frame.bc"
```